lib/customer.rb

```diff
@@ -24,7 +24,7 @@ def transactions
   end
 
   def favorite_merchant
-    successful_transactions = transactions.select { |transaction| transaction.result == "success" }
+    successful_transactions = transactions.select { |transaction| transaction.successful? }
```

Comment:
Line is too long. [91/80]<br>Pass `&:successful?` as an argument to `select` instead of a block.